examples/src/main/java/org/apache/ignite/examples/ml/tree/DecisionTreeClassificationTrainerExample.java

```diff
@@ -61,7 +61,7 @@ public static void main(String... args) throws InterruptedException {
                     trainingSet.put(i, generatePoint(rnd));
 
                 // Create classification trainer.
-                DecisionTreeClassificationTrainer trainer = new DecisionTreeClassificationTrainer(4, 0);
+                DecisionTreeClassificationTrainer trainer = new DecisionTreeClassificationTrainer(4, 0, false);
```

Comment:
Looks like it makes sense to add some default behavior (use index for example).

Reply:
Yes, it is. It may be fixed after Builder-like methods and "true"-value by default.